### ISSUE TITLE
Doc suggestion: "quick" start guide to qubes for non-technical specialists

### ISSUE THREAD

**Issue by mossy-nw** [Qubes-Community/Contents#60]
I'd like to start a "(relatively) quick-start guide" for non-technical specialists to setting up and running Qubes OS.  I don't have any systematic knowledge of the Qubes OS userbase, but it seems very likely to be quite limited to people reasonably comfortable with computers (or otherwise security enthusiasts willing to spend lots of time and ask lots of questions).  I would like to see Qubes OS benefits extend beyond this (however wonderful) user community, for the benefit of others but also for the benefit of the Qubes OS project itself (greater userbase means more testers, more contributors, more funders).

The documentation goal is an accessible guide to a maximally simplified qubes setup that takes shortcuts whenever possible but takes advantage of the many core compartmentalization benefits of qubes.  Basically the opposite of what many qubes users spend time doing (split-gpg with isolated master keys, etc.).  Any shortcuts should make life actually easier and of course be non-fatal (ideally minimally detrimental) to security.

The scope would be English language, Qubes R4.0+ and include everything from acquiring hardware to keeping on top of updates to suggestions for number and type and color codes for individual qube categories (the latter could be developed for numerous different kinds of users).

Any advice or things I should know (especially: is this a duplication of work?) before getting started would be appreciated!

Certainly I wonder if this duplicates anything (or would add to) what @freedomofpress or @liliakai is working on for journalists -- https://github.com/freedomofpress/securedrop-workstation 

Thanks everyone!

**Comment by SvenSemmler** [comment]
Hi @mossy-nw, my impression is that once Qubes is installed, it's actually quite easy to use after understanding the core concepts (compartmentalization, templates, disposable vms, clipboard and file transfer inter-qubes, assigning device, managing USB). If I understand you correctly, that's where you would like to improve documentation and make tutorials for the less technical user. This is certainly a good goal and I'd love to contribute.

However, I have some doubts about trying to promote Qubes to less technical users at this point in time. As the famous saying goes "you have only one first impression". In it's current state, installing Qubes is almost always a challenge. Even if you go with a ThinkPad or another Linux-friendly brand in almost all cases one needs to troubleshoot some technical issues before having a solid install. ... which is greatly helped by the super responsive dev team and the qubes-users community. However basic technical skills / understanding is required.

In summary, my personal opinion is that unless and until we have at least one reference notebook where Qubes can be simply installed and then works out of the box it would be a mistake to promote Qubes to the less technical. They will try, fail and then in most cases never consider Qubes again.

**Comment by one7two99** [comment]
Hello,

Am So., 30. Dez. 2018, 18:33 hat Sven Semmler <notifications@github.com>
geschrieben:

> [...] However, I have some doubts about trying to promote Qubes to less
> technical users at this point in time. [...]
>
I don't fully agree, as one argument I hear very often when talking about
privacy/security with "less technical users" is:
"(...) there is nothing I can do (...)"
I think we have to split those less technical users into two groups:
1) people who think that there are no alternatives
2) people who know that alternatives exist, but think they are not skilled
enough to try those alternatives.

A quickstart guide might help targeting both groups.

As the famous saying goes "you have only one first impression". In it's
> current state, installing Qubes is almost always a challenge. Even if you
> go with a ThinkPad or another Linux-friendly brand in almost all cases one
> needs to troubleshoot some technical issues before having a solid install.
>
I see two challenges with Qubes:
1) getting the right hardware
2) installing Qubes
3) understanding how Qubes works
4) migrating data from previous devices and services into Qubes

Step 1 can be easily solved by promoting certain laptops like the ThinkPads
or others brands.
Step 2 is also easy to cover and install is straight forward already.
Step 3 can also be covered in a small series of posts or videos.
Step 4 is the most difficult topic on my opinion.
Users should be able to do all tasks they're currently doing as it is
unlikely if they have to use two systems.

> In summary, my personal opinion is that unless and until we have at least
> one reference notebook where Qubes can be simply installed and then works
> out of the box it would be a mistake to promote Qubes to the less technical.
>
We should always promote Qubes so that users get appetite and then try it
out one day.
This is also how I have learned about Qubes.

- O


>

**Comment by Aekez** [comment]
I like your idea @mossy-nw, you have my encouragement for working on this project :)

Just as a reminder, although I'm sure everyone know this, though just in case as it sometimes can be misunderstood - we have no authority or insider knowledge among the Qubes OS staff, we are independent here at QCC. It is not our right to speak about whether Qubes should be promoted or not, this alone, imho, should be decided by the Qubes staff's strategic visions. Although as an volunteer like you, I do encourage to start discussing this over with the Qubes staff if you seek answers on growing the community, as their strategic vision about spreading Qubes OS to the masses, is indeed not very clear. Whether they want the community to try grow itself, when to grow, timing, how to grow, controlled or free growth, etc. there are a lot of (as far as I know) unanswered questions to the Qubes OS staff right there.

However, having said that, I do definitely agree with you on the need for making better material for consumption such as making better introductions and guides (which indirectly can grow the community). This we shouldn't need to ask the Qubes OS staff for, since this is outside the above unanswered strategic questions, and very much inside the Qubes staffs encouragement of any of its community members (i.e. free to contribute docs and code).

Taking a step back from needing the Qubes OS staff's view and permission on growing the community on our own (not legally, but out of respect), I think all parties in discussion so far has said something that makes a lot of sense, but seemingly opposing views needs to be pieced together to fit together without contradictions. Qubes OS is indeed easy for some, but it is also not easy for everyone. Even people savvy with Linux can have troubles with Qubes OS, while others who are beginners with Linux can have an easy time with Qubes OS. It's a bit weird like that, imho, but yet also understandable, people are different, and have different skills, such as being able to switch between domains, which comes easy to some people, but is very difficult for others (at least without first training it, which they might not have the patience for). Actually I think gamers might be good at learning Qubes OS, but that is only a random hypothetical speculation of mine, it is not on the level of theoretical or empirical evidence, and may be subject to confirmation bias. However if true, then gamers might actually be a target group, though perhaps not until it becomes possible to game on Qubes OS (I believe there are plans for it, but it's not solid, and I don't have insider knowledge other than public announced info). Until then though, perhaps some gamers might be willing to keep a gamer PC, while doing everything else on a Qubes OS system (i.e. desktop for gaming, but laptop with Qubes OS). Since gamers might learn Qubes OS easily, it could be an interesting target group, among gamers that care about security (which is granted not every gamer out there, who knows, maybe 5% or 10% of all gamers cares, but that is still quite a lot of people).

Another group could be security oriented people, I mean, only about 27.000 computer systems run Qubes OS right now for a couple of months, and 31.000 peak in the months when Qubes 4.0 was still very new. If needing the numbers, or curious, they can be found here https://www.qubes-os.org/statistics/ My point is, only about 27K~ people use Qubes OS, that is not a lot of people even among security focused people, taking into consideration how many people and organizations that lives on the internet. Making guides, perhaps pick the more savvy users first, before making future guides for the more average and normal users.

Qubes OS is undoubtedly becoming easier and easier to use too, i.e. the new update tool that made Qubes 4.0. much more tricky for people not savvy with the terminal, coming from Qubes 3.2., though, the new tool seems even better than the old. In general, there seems to be a lot of focus on UI improvement coming up, just by looking at the work being done lately. So how easy Qubes is to use, may change a lot in a few years, maybe even sooner.

Whatever you choose to do, and perhaps you're already thought about it, but my advice is to have a keen eye to your target group as you write guide or introduction materials (and maybe wait with some other target groups until later, that may only be more viable in a few years in the future, such as the masses), and also encourage to invite others to help you improve it (different views can be very useful when writing guides since every person is different, views the world and thinks differently). Finally, take your time to do this, if you want to write a good introduction guide, it won't be something you will do overnight, but probably more like something that evolves over time.

Just my humble views and opinions. Whatever you choose to do, I'm looking forward to see your work on your idea :)

**Comment by Aekez** [comment]
Another question you might want to seek answers on from the Qubes OS staff (extrapolating on my above post), is whether they feel like promoting Qubes OS among some types of gamers that care about security (which may especially become an interesting question once it becomes possible to game on Qubes OS). The reason is that Qubes OS staff may strive to focus for a security image and reputation, and having a lot of gamers flocking around (at least early on before getting more popular), may not be very desired. Perhaps they don't mind, perhaps they do, although personally I think it's better not to "hype" Qubes OS too early, and not to taint Qubes OS reputation by oversizing a population that many don't link with IT security, such as gamers. Personally, I think a healthy portion of Qubes OS members being gamers isn't a bad think, but I believe it mustn't grow too quickly, otherwise it may hype and taint the Qubes OS reputation. Who knows, it's hard to control such things, and by the end of the day, it's better to ask the Qubes OS staff before trying to pull in gamers. 

Then again, if wanting to quickly grow the user-base, some types of gamers might seem like a good group since grasping domains in their minds may come easy to them, but a quick growth may not necessarily be a good think either. These are hard to answer questions too, possibly hard for the Qubes OS staff too, since there quite a lot of unknowns in strategic decisions.

Aight, gonna stop here, hopefully you can use some of these thoughts :)

**Comment by Aekez** [comment]
Gotta admit, it is kinda embarrassing to miss-type assigning, tired + forgetting to enable noscript + doing it right after writing long post = not a good cocktail ~

**Comment by awokd** [comment]
Closing for now.
